flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/channel/ChannelStateSerializerImplTest.java

```diff
@@ -36,7 +36,6 @@
 import java.util.Random;
 
 import static org.apache.flink.runtime.checkpoint.channel.ChannelStateByteBuffer.wrap;
```

Comment:
Okay, now I see that you removed the original `ChannelStateReader`. I'd probably state that in the commit that adds `SeqReader`.

Reply:
Done.